includes/class-amp-theme-support.php

```diff
@@ -2114,6 +2100,7 @@ private static function get_optimizer_configuration( $args ) {
 				[
 					Optimizer\Transformer\AmpRuntimeCss::class,
 					Optimizer\Transformer\PreloadHeroImage::class,
+					Optimizer\Transformer\RewriteAmpUrls::class,
```

Comment:
Why was this added here? Disabling SSR should not have any effect on ESM, LTS or RTV, should it?

Reply:
This transformer requires `transformed` to be valid AMP. I added it here when trying to load a page with SSR disabled and it became invalid (and broken too).